cfconfiguration/cffinder_unix_test.go

```diff
@@ -0,0 +1,91 @@
+// +build !windows
+
+package cfconfiguration
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"os/user"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
TestMain function is used, when you want to do some operations before or after tests. In your case it can be skipped.

Reply:
Routine deleted.